### DISCUSSION TITLE
MacOS Keychain

### DISCUSSION THREAD

**Discussion by WebmasterOcto** [iterate-ch/cyberduck#16539] [Q&A]
I just downloaded Cyberduck. When I attempted my first connect, I went around in circles with the connection requesting my MacOS keychain password. I've been on MacOS for several years ans have no idea what the Keychain password is. How do I turn off Keychain? I prefer entering the passwords to my domains

**Comment by dkocher** [comment]
It is attempted to search for passwords in your login keychain. Refer to our [documentation](https://docs.cyberduck.io/cyberduck/connection/#passwords).

**Comment by tatze96** [comment]
Keychain usage can be deactivated within the Cyberduck preferences _Connection_ tab. You may need to remove items that are already present from the keychain database to ensure the subsequent password prompts work correctly.

**Comment by WebmasterOcto** [comment]
i deactivated keychain. I'm still getting the error. This seems to be an ongoing issue. How do I work around it?

Barney
-----Original Message-----
